src/database/migrations/0010-alter-application_forms-external_id.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE application_forms ADD COLUMN external_id VARCHAR;
```

Comment:
I notice that there is an `external_id` column on both `applicants` and `proposals`. The columns on those tables are both `NOT NULL` and `UNIQUE` (when combined with other columns, in the case of `proposals`). Why are these new `external_id`s nullable and not unique? Or, when would an `application_form` or an `application_form_field` not have an external ID?

Reply:
Currently they do not. So the application forms and form fields in the database today do not have it. Therefore we need some kind of default value, even if null.

Reply:
There probably exists an ID somewhere (in the external system) but that does not mean we have it. So we either need to delete and start over in order to constrain it to `NOT NULL` or allow `NULL`.

Reply:
This raises the question about whether every data platform provider will definitely have such an ID. I do not know the answer.